tests/zfs-tests/include/libtest.shlib

```diff
@@ -3278,8 +3278,7 @@ function zed_setup
 	log_must sed -i '/\#ZED_DEBUG_LOG=.*/d' $ZEDLET_DIR/zed.rc
 	echo "ZED_DEBUG_LOG=$ZEDLET_DIR/zed.debug.log" >>$ZEDLET_DIR/zed.rc
 
-	log_must cp ${ZEDLET_LIBEXEC_DIR}/all-syslog.sh $ZEDLET_DIR
-	log_must cp ${ZEDLET_LIBEXEC_DIR}/all-debug.sh $ZEDLET_DIR
+	log_must cp ${ZEDLET_LIBEXEC_DIR}/*.sh $ZEDLET_DIR
```

Comment:
If the test case requires additional zedlets be enabled let's do that in the specific test case.  For this one I don't believe any additional scripts are required since it's taken care of by the retire agent.

Reply:
Reverted this portion to the way it was before.